### PR TITLE
Completing AC 2 of 3005.

### PR DESCRIPTION
**Pull request by assoomro**

### CODE REVIEW

File: src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs [hunk 1]
```diff
@@ -14,6 +14,16 @@ public PreviewPage(IWebDriver driver, ITestOutputHelper helper) : base(driver, h
         {
         }
 
+        /// <summary>
+        /// Awful, will get replaced with a proper method once preview page contains a way to go to dashboard.
+        /// </summary>
+        public void NavigateToDashboard()
```
Comment (PhysicsJon) [inline src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs]: Could be replaced with 'driver.Navigate().Back()`

Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs]: Would then be good to wait for something found on the dashboard to be displayed after calling the `Back()` method

Comment (assoomro) [reply src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs]: Good idea, done and done.

File: src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/PageAction.cs [hunk 2]
```diff
@@ -28,6 +28,10 @@ public PageAction(IWebDriver driver, ITestOutputHelper helper)
             // Initialize the page objects
             pages = new PageObjects().Pages;
         }
-        
+        public void GoBackOnePage()
```
Comment (PhysicsJon) [inline src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/PageAction.cs]: I think this should go in the Common class (it's likely to be used in a lot of places

Comment (assoomro) [reply src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/PageAction.cs]: Good point, done.

**Pull request merged**